packages/svelte/package.json

```diff
@@ -54,8 +54,8 @@
   },
   "devDependencies": {
     "@testing-library/svelte": "^3.2.1",
-    "svelte": "3.49.0",
-    "svelte-jester": "^2.3.2"
+    "@sveltejs/vite-plugin-svelte": "latest",
```

Comment:
let's always hard pin to something, otherwise we risk not having deterministic CI.

Reply:
do you have a version in mind?

Reply:
pinned 3.0.1